core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -873,6 +873,16 @@
   public static final String TABLE_BLOCKSIZE = "table_blocksize";
   // set in column level to disable inverted index
   public static final String NO_INVERTED_INDEX = "no_inverted_index";
+  // table property name of major compaction size
+  public static final String MAJOR_COMPACTION_SIZE = "major_compaction_size";
```

Comment:
For these compaction properties of Table level , suggest adding "TABLE", such as : TABLE_MAJOR_COMPACTION_SIZE,TABLE_AUTO_LOAD_MERGE...